package.json

```diff
@@ -117,6 +117,13 @@
       "default": true,
       "order": 6
     },
+    "enableLatexmkrc": {
```

Comment:
I like those, but what about something even more verbose, e.g. `enableExtendedBuildRules`? Perhaps even avoiding the somewhat ambiguous "rules" term with something like `enableExtendedBuildMode` or `enableExtendedBuild`?